k8s/perftest/common/draft-store/draft-store-service.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: draft-store-service
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: draft-store-service
+    version: 1.0.3
+  values:
+    java:
```

Comment:
shouldn't this be set @adusumillipraveen ?
`useInterpodAntiAffinity: true`

and all the pdb flags?
https://github.com/hmcts/chart-java

Reply:
Yes, AAT is better to refer : https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/rpe/draft-store/draft-store-service.yaml

Reply:
even AAT is missing pdb 👅

Reply:
PDB is set to 50% by default.

Reply:
ah I missed that

Reply:
done..